docs/src/pages/guides/typescript.md

```diff
@@ -94,6 +94,42 @@ const DecoratedNoProps = decorate<{}>( // <-- note the type argument!
 
 To avoid worrying about this edge case it may be a good habit to always provide an explicit type argument to `decorate`.
 
+### Injecting Multiple Classes
+
+Injecting multiple classes into a component is straightforward possible. Take the following code for example: The classes `one` and `two` are both available with type information on the `classes`-prop passed in by `withStyles`.
```

Comment:
"is *as* straightforward *as* possible" (add 2 x "as")

Regarding "Take the following code for example:" it should be something other than a colon here, as the example doesn't immediately follow, but I can't decide if it should be a full stop (period), or a hyphen, running on to the next clause. Take your pick, either will work. 

Sorry for being picky. 😈

Reply:
No problem. Just please no more edits.

Reply:
Or just edit the PR directly. :D